CMakeLists.txt

```diff
@@ -1,6 +1,8 @@
-#  minimum required cmake version: 3.1.0
+
+# Xionic requires 3.5, Bionic 3.10...
```

Comment:
I meant here about the "Xionic" > "Xenial".
When I added a comment on a specific commit it add it as a general comment somehow..

Reply:
Right :)
Fixed

Reply:
😀 Great, see you tomorrow